pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -272,7 +274,12 @@ func NewLegacyRegistry() *LegacyRegistry {
 		})
 	registry.registerPredicateConfigProducer(CheckVolumeBindingPred,
 		func(args ConfigProducerArgs) (plugins config.Plugins, pluginConfig []config.PluginConfig) {
+			pluginConfig = append(pluginConfig, NewPluginConfig(volumebinding.Name, args.VolumeBindingArgs))
```

Comment:
I'm removing `NewPluginConfig` in #90309
This will be merged soon.

However, the `LegacyRegistry` is only intended for args that are affected by the old Policy API. This is not the case for bind timeout, so we shouldn't add the type here. Just generate the config in factory.